### PR TITLE
Treat messages where fatal is set to null as 'info'

### PR DESCRIPTION
**Pull request by brendo**
Related to https://github.com/vfile/vfile/pull/18, in fact the test can be cleaned up a little once `isFatal` can be set through the message constructor.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -5,11 +5,11 @@ module.exports = statistics;
 /* Get stats for a file, list of files, or list of messages. */
 function statistics(files) {
   var total = 0;
-  var result = {true: 0, false: 0};
+  var result = {true: 0, false: 0, null: 0};
 
   count(files);
 
-  return {fatal: result.true, nonfatal: result.false, total: total};
+  return {fatal: result.true, nonfatal: result.false, info: result.null, total: total};
```
Comment (wooorm) [inline index.js]: How about adding `'warn'` too? And using nonfatal as all values that are not `fatal: true` (the current behaviour)?

File: index.js [hunk 2]
```diff
@@ -33,7 +33,7 @@ function statistics(files) {
       length = value.length;
 
       while (++index < length) {
-        result[Boolean(value[index].fatal)]++;
+        result[value[index].fatal]++;
```
Comment (wooorm) [inline index.js]: `fatal: undefined` should be treated as `null` / `info`

File: readme.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 # vfile-statistics [![Build Status][travis-badge]][travis] [![Coverage Status][codecov-badge]][codecov]
 
-Count [vfile][] messages per category (fatal, nonfatal, and total).
+Count [vfile][] messages per category (fatal, nonfatal, info and total).
```
Comment (wooorm) [inline readme.md]: and `warn`

File: readme.md [hunk 4]
```diff
@@ -31,7 +31,7 @@ console.log(statistics(file));
 Yields:
 
 ```js
-{ fatal: 1, nonfatal: 2, total: 3 }
+{ fatal: 1, nonfatal: 2, info: 0, total: 3 }
```
Comment (wooorm) [inline readme.md]: and `warn`, and could you add an info message so it counts as `info: 1`?

**Review by wooorm — CHANGES_REQUESTED** [review]

**Comment by brendo** [comment]
I've made changes based of your review. The readme example could be a cleaned up a little (and the test) once `.info` is available on vfile. Happy to do that once that happens 👍

**Review by wooorm — APPROVED** [review]

**Comment by wooorm** [comment]
I refactored it a bit, but don’t take that personally. Thanks a lot for working on this! 🎉

**Pull request closed without merging**